libfreerdp/core/window.c

```diff
@@ -498,6 +497,148 @@ static BOOL window_order_supported(const rdpSettings* settings, UINT32 fieldFlag
 	}
 }
 
+
+#define DUMP_APPEND(...) do { \
+		snprintf(tmpBuffer, sizeof(tmpBuffer), __VA_ARGS__); \
+		strncat(buffer, tmpBuffer, rem); \
+		rem -= strlen(tmpBuffer); \
+		} while(0)
+
+
+static void dump_window_state_order(wLog *log, const char *msg, const WINDOW_ORDER_INFO* order, const WINDOW_STATE_ORDER *state)
+{
+	char buffer[3000];
```

Comment:
a `= { 0 };` might be a good idea

Reply:
Not needed, the first call is a snprintf, that already adds the  ̀\0`